docs/docs/users/reference/env_variables.md

```diff
@@ -31,6 +31,7 @@ process.
 | `FOREST_TEST_RNG_FIXED_SEED`                              | non-negative integer             | empty                                          | 0                                                             | Override RNG with a reproducible one seeded by the value. This should never be used out of test context for security. |
 | `RUST_LOG`                                                | string                           | empty                                          | `debug,forest_libp2p::service=info`                           | Allows for log level customization.                                                                                   |
 | `FOREST_IGNORE_DRAND`                                     | 1 or true                        | empty                                          | 1                                                             | Ignore Drand validation.                                                                                              |
+| `FOREST_LIBP2P_METRICS`                                   | 1 or true                        | empty                                          | 1                                                             | Include `libp2p` metrics in Forest's Prometheus output.                                                               |
```

Comment:
Looking at other variables (at least newer ones), it seems the convention is to have a verb to make it more obvious,e.g., `_OPT_OUT` (here rather opt-in), `_ENABLED` etc.

Reply:
Added `_ENABLED`